### DISCUSSION TITLE
Notification over fullscreen element

### DISCUSSION THREAD

**Discussion by imirzadeh** [quasarframework/quasar#9512] [General - Components / Directives / etc]
Hi,

I'm using video.js to play videos and I would like to show some notifications when the video is in full-screen.
However, when the video is in the full-screen mode, the notification goes behind the video as you can see in the picture:
![image](https://user-images.githubusercontent.com/8312051/120399520-0616da80-c2f1-11eb-8dba-1bd79489ddca.png)

But when the video is not in the full-screen mode, it shows on top of the video
![image](https://user-images.githubusercontent.com/8312051/120399609-38283c80-c2f1-11eb-9c0f-893fb451d2b1.png)

Is there any way to show a notification when an element is in full-screen mode? For example, is there a way to force the notification to be always on top of other elements?

**Comment by ontwikkelfabriek** [comment]
@imirzadeh 

It's possible using Quasar's QMediaPlayer.

Here's an example with an overlay that will display also in fullscreen mode:
https://quasarframework.github.io/quasar-ui-qmediaplayer/examples#title-Bottom-Controls

QmediaPlayer docs:
https://quasarframework.github.io/quasar-ui-qmediaplayer/docs

**Reply by imirzadeh** [reply]
Thanks for the reply. I wish I could use QMediaPlayer but unfortunately it doesn't support HLS/DASH streaming.

**Reply by ontwikkelfabriek** [reply]
@imirzadeh 

Ah I see, if you create a codepen.io with your video player and a q-notify we could take a look at your problem.
